angular-flot.js

```diff
@@ -151,6 +156,9 @@ angular.module('angular-flot', []).directive('flot', ['$timeout', function ($tim
       //
 
       element.on('$destroy', function onDestroy () {
+        
+        $(window).off('resize'); // remove the handler added earlier
```

Comment:
I think the worst is not the comment but the fact that this would unbind all `resize` handlers on the window